### DISCUSSION TITLE
1.1.1

### DISCUSSION THREAD

**Discussion by Eldritch-Oliver** [Eldritch-Oliver/file-hider#28] [Announcements]
## What's Changed
* Fixed files not hiding on obsidian initial load by @KosmosisDire in https://github.com/Oliver-Akins/file-hider/pull/27

**Full Changelog**: https://github.com/Oliver-Akins/file-hider/compare/1.1.0...1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Oliver-Akins/file-hider/releases/tag/1.1.1'>1.1.1</a>.</em>

**Comment by KosmosisDire** [comment]
@Oliver-Akins Hey thanks! However the update is not detected by obsidian. This is because the `manifest.json` is pulled from the main branch rather than the release, and the version on the main branch is still set to version 1.1.0

**Reply by Eldritch-Oliver** [reply]
Hahah, whoops, I've updated the in-repo manifest now. Thanks for letting me know I missed it, it's been a while since I published a release for it.
